### ISSUE TITLE
Building Error on Windows of master branch

### ISSUE THREAD

**Issue by Enigma-li** [tensorflow/tensorflow#12332]
After fixing the error mention in #12288 , there are new error as shown below:

```
.....................................
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_event.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_fft.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_gpu_executor.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_platform.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_platform_id.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_rng.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_stream.obj"
"C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_stream_executor.dir\Release\cuda_timer.obj"
Creating library C:/v-clhian/tensorflow/tensorflow/contrib/cmake/build/Release/pywrap_tensorflow_internal.lib and object C:/v-clhian/tensorflow/tensorflow/contrib/cmake/build/Release/pywrap_tensorflow_internal.exp
5>pywrap_tensorflow_internal.exp : warning LNK4070: /OUT:_pywrap_tensorflow_internal.pyd directive in .EXP differs from output filename 'C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\Release\pywrap_tensorflow_internal.dll'; ignoring directive [C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\pywrap_tensorflow_internal.vcxproj]
5>zero_initializer_op.obj : error LNK2019: unresolved external symbol "public: void __cdecl tensorflow::functor::TensorSetZero<struct Eigen::GpuDevice,struct Eigen::half>::operator()(struct Eigen::GpuDevice const &,class Eigen::TensorMap<class Eigen::Tensor<struct Eigen::half,1,1,__int64>,16,struct Eigen::MakePointer>)" (??R?$TensorSetZero@UGpuDevice@Eigen@@uhalf@2@@functor@tensorflow@@qeaaxaebugpudevice@Eigen@@v?$TensorMap@V?$Tensor@Uhalf@Eigen@@$00$00_J@Eigen@@$0BA@UMakePointer@2@@4@@z) referenced in function "public: virtual void __cdecl tensorflow::ZeroInitializerOp<struct Eigen::GpuDevice,struct Eigen::half>::Compute(class tensorflow::OpKernelContext *)" (?Compute@?$ZeroInitializerOp@UGpuDevice@Eigen@@uhalf@2@@tensorflow@@ueaaxpeavopkernelcontext@2@@z) [C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\pywrap_tensorflow_internal.vcxproj]
5>zero_initializer_op.obj : error LNK2019: unresolved external symbol "public: void __cdecl tensorflow::functor::TensorSetZero<struct Eigen::GpuDevice,float>::operator()(struct Eigen::GpuDevice const &,class Eigen::TensorMap<class Eigen::Tensor<float,1,1,__int64>,16,struct Eigen::MakePointer>)" (??R?$TensorSetZero@UGpuDevice@Eigen@@m@functor@tensorflow@@qeaaxaebugpudevice@Eigen@@v?$TensorMap@V?$Tensor@M$00$00_J@Eigen@@$0BA@UMakePointer@2@@4@@z) referenced in function "public: virtual void __cdecl tensorflow::ZeroInitializerOp<struct Eigen::GpuDevice,float>::Compute(class tensorflow::OpKernelContext *)" (?Compute@?$ZeroInitializerOp@UGpuDevice@Eigen@@m@tensorflow@@ueaaxpeavopkernelcontext@2@@z) [C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\pywrap_tensorflow_internal.vcxproj]
5>zero_initializer_op.obj : error LNK2019: unresolved external symbol "public: void __cdecl tensorflow::functor::TensorSetZero<struct Eigen::GpuDevice,double>::operator()(struct Eigen::GpuDevice const &,class Eigen::TensorMap<class Eigen::Tensor<double,1,1,__int64>,16,struct Eigen::MakePointer>)" (??R?$TensorSetZero@UGpuDevice@Eigen@@n@functor@tensorflow@@qeaaxaebugpudevice@Eigen@@v?$TensorMap@V?$Tensor@N$00$00_J@Eigen@@$0BA@UMakePointer@2@@4@@z) referenced in function "public: virtual void __cdecl tensorflow::ZeroInitializerOp<struct Eigen::GpuDevice,double>::Compute(class tensorflow::OpKernelContext *)" (?Compute@?$ZeroInitializerOp@UGpuDevice@Eigen@@n@tensorflow@@ueaaxpeavopkernelcontext@2@@z) [C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\pywrap_tensorflow_internal.vcxproj]
5>C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\Release\pywrap_tensorflow_internal.dll : fatal error LNK1120: 3 unresolved externals [C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\pywrap_tensorflow_internal.vcxproj]
5>Done Building Project "C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\pywrap_tensorflow_internal.vcxproj" (default targets) -- FAILED.
1>Done Building Project "C:\v-clhian\tensorflow\tensorflow\contrib\cmake\build\tf_python_build_pip_package.vcxproj" (default targets) -- FAILED.

Build FAILED.
```

May be some other Cmakelist error, could you help fix this?

**Comment by mrry** [comment]
This should be fixed by #12315. Can you try pulling that branch and see if it fixes the problem for you (you might get a result faster than waiting for our CI builds :)...)?

**Comment by Enigma-li** [comment]
Thanks, I will try again.
